molo/commenting/tests/test_models.py

```diff
@@ -126,74 +125,3 @@ def test_auto_remove_for_non_remove_flag(self):
             )
         altered_comment = MoloComment.objects.get(pk=comment.pk)
         self.assertFalse(altered_comment.is_removed)
-
-
-class CommentingSettingsTest(TestCase, MoloTestCaseMixin):
-    """Test if the anonymous commengting alias value is set properly."""
-
-    def setUp(self):
-        self.mk_main()
-        self.main = Main.objects.all().first()
-        self.language_setting = Languages.objects.create(
-            site_id=self.main.get_site().pk)
-        self.english = SiteLanguageRelation.objects.create(
-            language_setting=self.language_setting,
-            locale='en',
-            is_active=True)
-        self.user = User.objects.create_user(
-            'test', 'test@example.org', 'test')
-        self.content_type = ContentType.objects.get_for_model(self.user)
-        self.yourmind = self.mk_section(
-            self.section_index, title='Your mind')
-        self.article1 = self.mk_article(
-            title='article 1', slug='article-2', parent=self.yourmind)
-        self.setting = CommentingSettings.for_site(self.site)
-
-        self.french = SiteLanguageRelation.objects.create(
-            language_setting=Languages.for_site(self.site),
-            locale='fr',
-            is_active=True)
-        self.translated_article = self.mk_article_translation(
-            self.article1, self.french,
-            title=self.article1.title + ' in french',
-            subtitle=self.article1.subtitle + ' in french')
-
-    def test_get_comments_anonymous(self):
```

Comment:
I think you can leave this test (and whatever setup is required for it) here. It's good to have